ios/NotifeeCore/NotifeeCore+UNUserNotificationCenter.m

```diff
@@ -104,20 +104,28 @@ - (void)userNotificationCenter:(UNUserNotificationCenter *)center
       presentationOptions |= UNNotificationPresentationOptionSound;
     }
 
-    if (alert) {
-      presentationOptions |= UNNotificationPresentationOptionAlert;
-    }
-
-    if (banner) {
-      if (@available(iOS 14, *)) {
-        presentationOptions |= UNNotificationPresentationOptionBanner;
-      }
-    }
+    // if list or banner is true, ignore alert property
```

Comment:
Here is the result of `yarn format:all`. I could not push it due to permission tho.

```diff
     // if list or banner is true, ignore alert property
     if (banner || list) {
-        if (banner) {
-          if (@available(iOS 14, *)) {
-            presentationOptions |= UNNotificationPresentationOptionBanner;
-          } else {
-              // for iOS 13 we need to set alert
-            presentationOptions |= UNNotificationPresentationOptionAlert;
-          }
+      if (banner) {
+        if (@available(iOS 14, *)) {
+          presentationOptions |= UNNotificationPresentationOptionBanner;
+        } else {
+          // for iOS 13 we need to set alert
+          presentationOptions |= UNNotificationPresentationOptionAlert;
         }
-
-        if (list) {
-          if (@available(iOS 14, *)) {
-            presentationOptions |= UNNotificationPresentationOptionList;
-          } else {
-              // for iOS 13 we need to set alert
-            presentationOptions |= UNNotificationPresentationOptionAlert;
-          }
+      }
+
+      if (list) {
+        if (@available(iOS 14, *)) {
+          presentationOptions |= UNNotificationPresentationOptionList;
+        } else {
+          // for iOS 13 we need to set alert
+          presentationOptions |= UNNotificationPresentationOptionAlert;
         }
+      }
     } else if (alert) {
-        // TODO: remove alert once it has been fully removed from the notifee API
-        presentationOptions |= UNNotificationPresentationOptionAlert;
+      // TODO: remove alert once it has been fully removed from the notifee API
+      presentationOptions |= UNNotificationPresentationOptionAlert;
     }
```

Reply:
hadn't run the formatter yet, good catch 🙏